fp.js

```diff
@@ -18,9 +26,16 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(person => {
+//people.forEach(person => {
+//  if(person.age < 35){
+//    peopleYoungerThan35.push(person)
+//  }
+//})
+
+//This is my answer to question #2
+var newFilter35 = people.filter(person => {
   if(person.age < 35){
-    peopleYoungerThan35.push(person)
+    return person.age;
   }
 })
```

Comment:
Close. A couple notes...

* The result of the original question is an array of people under 35, not their ages (i.e., it's an array of objects).
* While your `filter` does produce an array of ages, this is not how `filter` is used. The return value of `filter` should be the condition that you want each item in the final array to satisfy. So in this case, where each person should be under 35, `filter` will look something like this...

```js
const peopleUnder35 = people.filter(person => {
  // everybody in peopleUnder35 should match the below condition
  return person.age < 35;
}
```